src/tools/genpolicy/rules.rego

```diff
@@ -960,17 +960,10 @@ allow_storages(p_storages, i_storages, bundle_id, sandbox_id) {
 
     p_count == i_count
 
-    # Get the container image layer IDs and verity root hashes, from the "overlayfs" storage.
-    some overlay_storage in p_storages
-    overlay_storage.driver == "overlayfs"
-    print("allow_storages: overlay_storage =", overlay_storage)
-    count(overlay_storage.options) == 2
-
-    layer_ids := split(overlay_storage.options[0], ":")
-    print("allow_storages: layer_ids =", layer_ids)
+    layer_ids := []
+    root_hashes := []
```

Comment:
I feel this is altering the flow a bit as these will be empty, even if `enable_image_layer_storage` is enabled. Maybe we can have `allow_container_image_storage` fill and return `layer_ids` and `root_hashes`? if `enable_image_layer_storage` is enabled, fills them out. Otherwise, leaves them empty